test/unit/job_invocation_test.rb

```diff
@@ -0,0 +1,44 @@
+require 'test_plugin_helper'
+
+describe JobInvocation do
+
+  let(:job_invocation) { FactoryGirl.build(:job_invocation) }
+  let(:template) { FactoryGirl.build(:job_template) }
+
+  context 'Able to be created' do
+    it { assert job_invocation.save }
```

Comment:
I prefer at least one real db save otherwise we're just assuming the schema is valid which it may not be.